src/Glpi/Marketplace/View.php

```diff
@@ -268,6 +268,44 @@ public static function installed(
         self::displayList($plugins, "installed", $only_lis);
     }
 
+    /**
+     * Get the logo of a plugin
+     * Useful for custom plugins which will be used internally only and are not published in the internet
+     *
+     * @param array   $apidata  collected data of the plugin
+     * @param string  $key      plugins directory name
+     *
+     * @return string url to the plugins logo or empty string
+     */
+    public static function getLogo(array $apidata, string $key): string
+    {
+
+        // found a logo from online resource?
+        if (isset($apidata['logo_url']) && strpos(strtolower($apidata['logo_url']), '.png') !== false) return $apidata['logo_url'];
```

Comment:
IMHO, it would even be preferable to serve the local icon first. Some GLPI administrators complains about the fact that the marketplace view requires access to remote contents.